tests/rados/test_osd_args.py

```diff
@@ -0,0 +1,77 @@
+from ceph.ceph_admin import CephAdmin
```

Comment:
Add a very brief summary of the module intent

Reply:
Added the module intent.